bcipy/helpers/acquisition.py

```diff
@@ -165,7 +165,7 @@ def init_lsl_client(parameters: dict,
     """Initialize a client that acquires data from LabStreamingLayer."""
 
     data_buffer_seconds = round(max_inquiry_duration(parameters))
-
+    log.info(f"Setting an acquisition buffer of {data_buffer_seconds} seconds")
```

Comment:
It might be helpful to know the device this applies to. Right now this logs twice with the same value.

Reply:
Good suggestion